### PR TITLE
Adopt --code-font-family

### PR DESCRIPTION
**Pull request by KTibow**
## Proposed change

This makes more cards use the new `--code-font-family` variable for their `<pre>` tags.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

*Kinda both... The feature isn't new, I've added this for the code editor before, so it fixes the bug of partial adoption, but it could also be that it's making the new feature complete.*

## Example configuration

```yaml
frontend:
  themes:
    comicsanscode:
      code-font-family: Comic Sans MS
```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by ludeeus** [comment]
You can skip some places by adding it to `ha-markdown` instead.

**Comment by KTibow** [comment]
How could I do that?

**Comment by ludeeus** [comment]
```
diff --git a/src/components/ha-markdown.ts b/src/components/ha-markdown.ts
index 978a9797..484776a2 100644
--- a/src/components/ha-markdown.ts
+++ b/src/components/ha-markdown.ts
@@ -72,6 +72,7 @@ class HaMarkdown extends LitElement {
         padding: 16px;
         overflow: auto;
         line-height: 1.45;
+        font-family: var(--code-font-family, monospace);
       }
       ha-markdown-element h2 {
         font-size: 1.5em;

```

**Comment by KTibow** [comment]
Thanks for the tip. Now how can I remove redundant code?

**Comment by ludeeus** [comment]
The same way you added it?

**Comment by KTibow** [comment]
I don't want to remove it in places where it's actually needed.

**Comment by KTibow** [comment]
Manually looked through all files, couldn't find any places where I could skip it.

File: hassio/src/system/hassio-host-info.ts [hunk 1]
```diff
@@ -413,6 +413,9 @@ class HassioHostInfo extends LitElement {
         mwc-list-item ha-svg-icon {
           color: var(--secondary-text-color);
         }
+        pre {
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts:416]: this is inserted to ha-markdown

File: src/panels/config/integrations/integration-panels/zwave/zwave-log-dialog.js [hunk 2]
```diff
@@ -15,6 +15,11 @@ class ZwaveLogDialog extends EventsMixin(PolymerElement) {
         <h2>OpenZwave internal logfile</h2>
         <paper-dialog-scrollable>
           <pre>[[_ozwLog]]</pre>
+          <style>
+            pre {
+              font-family: var(--code-font-family, monospace);
+            }
```
Comment (ludeeus) [inline src/panels/config/integrations/integration-panels/zwave/zwave-log-dialog.js:21]: Style is already on line 12-13, reuse that.

**Review by ludeeus — APPROVED** [review]

**Pull request merged**